### ISSUE TITLE
Projects using FCC test suite fail accessibility audits due to insufficient contrast ratio

### ISSUE THREAD

**Issue by abhishekakade** [freeCodeCamp/testable-projects-fcc#874]
#### Issue Description

Projects using FCC test suite fail accessibility audits due to insufficient color contrast on the buttons "Run Tests" and "Tests". The following values used to style the buttons in `src/stylesheets/fcc-test-ui.css` are the cause of this issue - 
```
button.fcc_foldout_button {
  background-color: rgba(128, 128, 128, 0.7);
  color: white;
}
```

#### Browser Information

* Browser Name, Version: Chrome Version 73.0.3683.103 (Official Build) (64-bit)
* Operating System: Windows 10 Pro
* Mobile, Desktop, or Tablet: Desktop

#### Your Code / Link to Your Pen

Go to this link - [Markdown Previewer](https://abhishekakade.github.io/markdown-previewer/) and run Audits from Chrome with Accessibility features checked to see the issue.
```
 // this is the suggested fix
button.fcc_foldout_button {
  background-color: rgba(50, 50, 50, 0.75);
  color: white;
}
 
```

#### Screenshot

[fcc-test-suite-accessibility-issue-ss.jpg](https://i.postimg.cc/c4gHdByh/fcc-test-suite-accessibility-issue-min.jpg)
